torch_xla/stablehlo.py

```diff
@@ -262,6 +280,11 @@ def run_node(self, n) -> Any:
         ]
         self._mark_dynamic(res, dynamic_dims)
       return res
+    if n.op == 'call_function' and n.target in custom_ops_allowed_in_graph:
```

Comment:
..then this like becomes `... and n.target.namespace in custom_ops_allowed_in_graph` etc